build.gradle

```diff
@@ -7,7 +7,7 @@ buildscript {
         mavenCentral()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:7.3.0'
+        classpath 'com.android.tools.build:gradle:7.2.2'
```

Comment:
Careful with this downgrade, maybe a bad merge? Remove this change, please.

Reply:
Don't know how that got there, is fixed now